pkgs/development/python-modules/pyinstaller-hooks-contrib/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+}:
+
+buildPythonPackage rec {
+  pname = "pyinstaller-hooks-contrib";
+  version = "2023.1";
+
+  format = "setuptools";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "ab56c192e7cd4472ff6b840cda4fc42bceccc7fb4234f064fc834a3248c0afdd";
+  };
+
+
+  doCheck = false;
+
+  meta = with lib; {
+    description = "PyInstaller communit hooks";
```

Comment:
```suggestion
    description = "PyInstaller community hooks";
```

Reply:
Will do when I am home and able to squash the commits for this change.